x-pack/plugins/security_solution/cypress/integration/exceptions/from_alert.spec.ts

```diff
@@ -28,20 +27,25 @@ import {
 import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../urls/navigation';
 import { cleanKibana, deleteAlertsAndRules } from '../../tasks/common';
 
-describe.skip('From alert', () => {
+describe('From alert', () => {
   const NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS = '1 alert';
 
   before(() => {
     cleanKibana();
     login();
   });
+
   beforeEach(() => {
-    esArchiverLoad('auditbeat_for_exceptions');
+    esArchiverLoad('exceptions');
     deleteAlertsAndRules();
-    createCustomRule({ ...getNewRule(), index: ['exceptions-*'] }, 'rule_testing');
+    createCustomRule(
```

Comment:
is there any reason why `createCustomRuleEnabled` was not used? If we are now passing as an argument the activation of the rule, I suggest refactoring the test suite in order to delete `createCustomRuleEnabled` and avoid maintaining 2 methods that do the same.